Tests/ViewInspectorTests/Gestures/SpatialTapGestureTests.swift

```diff
@@ -0,0 +1,144 @@
+import XCTest
+import SwiftUI
+import Combine
+@testable import ViewInspector
+
+// MARK: - Spatial Tap Gesture Tests
+
+@available(iOS 16.0, macOS 13.0, watchOS 9.0, *)
+@available(tvOS, unavailable)
+final class SpatialTapGestureTests: XCTestCase {
```

Comment:
These were largely pilfered from RotationGestureTests. I'm not sure I understand the full scope of CommonGestureTests, but I hope this covers all intended uses of SpatialTapGesture.